files/js.js

```diff
@@ -0,0 +1,161 @@
+$(".inscription").click(function(){
+    $(".match_container").show();
+    $(".slack_container").hide();
+});
+
+$(".slackusers").click(function(){
+    $(".match_container").hide();
+    $(".slack_container").show();
+    pop();
+});
+
+$.ajaxSetup({
+    async: false
+});
+
+let slackUsers = null;
+$.getJSON("getSlackUsers.php", function( data ) {
+    slackUsers = data["members"];
+});
+
+function userprofileName(user) {
+    var name = "";
+    
+    if(user.profile.hasOwnProperty("first_name") && user.profile.first_name.length > 0 && 
+       user.profile.hasOwnProperty("last_name")  && user.profile.last_name.length > 0) {
+	name += user.profile.first_name + " " + user.profile.last_name;
+    } else if(user.profile.hasOwnProperty("real_name") && user.profile.real_name.length > 0) {
+	name += user.profile.real_name;
+    } else if(user.profile.hasOwnProperty("display_name") && user.profile.display_name.length > 0) {
+	name += user.profile.display_name;
+    } else if(user.hasOwnProperty("real_name")  && user.real_name.length > 0) {
+	name += user.real_name;
+    } else if(user.hasOwnProperty("display_name") && user.display_name.length > 0) {
+	name += user.display_name;
+    }
+
+    return name;
+}
+
+function pop() {
+    var slack_IDS = [];
+    $(".matched[slackID]").each(function(){
+	slack_IDS.push(($(this).attr("slackID")));
+    });
+    console.log(slack_IDS);
+    $(".slack_container tbody").children().remove();
+    slackUsers.forEach((user) => {
+	console.log(user.id);
+	if(!user.deleted && !slack_IDS.includes(user.id) && !user.is_bot && ! user.is_app_user && user.id !== "USLACKBOT") {
+	    var row = $(`<tr><td>${user.id}</td><td>${userprofileName(user)}</td></tr>`).appendTo($($(".slack_container tbody")[0]));
```

Comment:
it's a detail, but could you help me understand why we need this variable `row`? (IIUC we don't use it afterwards, especially since we're in a loop here)

Reply:
Fixed in #f79bc35